Makefile

```diff
@@ -252,16 +252,19 @@ else
 buildx:
 endif
 
+BUILDX_COMMIT ?= v0.3.0
+export BUILDX_COMMIT
+
 bundles/buildx: BUILDX_DOCKERFILE ?= Dockerfile.buildx
-bundles/buildx: BUILDX_COMMIT ?= v0.3.0
 bundles/buildx: bundles ## build buildx CLI tool
 	# This intetionally is not using the `--output` flag from the docker CLI which is a buildkit option
 	# The idea here being that if buildx is being used, it's because buildkit is not supported natively
-	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$(BUILDX_COMMIT)" \
+	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$${BUILDX_COMMIT:-latest}" \
 		--build-arg BUILDX_COMMIT \
 		--build-arg BUILDX_REPO \
 		--build-arg GOOS=$$(if [ -n "$(GOOS)" ]; then echo $(GOOS); else go env GOHOSTOS || uname | awk '{print tolower($$0)}' || true; fi) \
 		--build-arg GOARCH=$$(if [ -n "$(GOARCH)" ]; then echo $(GOARCH); else go env GOHOSTARCH || true; fi) \
 		. && \
-		id=$$(docker create moby-buildx:$(BUILDX_COMMIT)); \
-		if [ -n "$${id}" ]; then docker cp $${id}:/usr/bin/buildx $@ && touch $@; docker rm -f $${id}; fi
+		id=$$(docker create moby-buildx:$${BUILDX_COMMIT:-latest}); \
```

Comment:
OK, so using `&&` from makefile is redundant. If any command fails, make stops right there. I guess this practice comes from Dockerfile `RUN` use, but makefile is different and if we can make things simpler, let's do it.

In other words,
```
	docker build ....
	id=$$(docker ...); if [ -n "$${id}" ]; ....
	$@ version
```

Reply:
Ok I rewrote it a bit; also put the `if` over a couple of lines, but let me know if that's too verbose.